### ISSUE TITLE
Comments on results 09/13

### ISSUE THREAD

**Issue by gaow** [mgrasso24/dyslexia-omics-integration#5]
@mgrasso24 it is great you finished the first draft of analysis! Here are some quick comments on the analysis:

1. Please display the figures direction inside the notebook instead of separately, so the notebook becomes self-contained. 
2. Please reconsider the organization of the notebook. It's missing important details upfront such as input data format (which you have struggled with). As a reminder please check out [this page](https://wanggroup.org/orientation/reproducible_research.html) on suggested reproducible research practice in the lab (notice here I prefer conclusion to precede detailed analysis codes so it's very clear what take home messages are. 
3. there is a band on the eQTL data well above 1E-30, on the top of the plot. What are those variants? It looks weird. Could you clarify or fix?
Thanks!

**Comment by mgrasso24** [comment]
Ok, I finished all of these notes! Thank you for checking my notebook, let me know if there are any other points you have noticed that I should fix!

**Comment by gaow** [comment]
Thanks. Still the conclusion seems to be "everything is eQTL" and not particularly about your top variants in GWAS. Remember that I asked you to try different thresholds for eQTL until you think the message is clear. Could you try 1E-30 see what it looks like? There is not much of a difference between 1E-5 and 1E-7 so we dont have to show 1E-7 if you showed 1E-5 already.

Additional comments:

1. I see you have improved formatting. Could you further improve it by using markdown section formatter? For example:

```
# title

## section 1

## section 2
```

2. The question regarding the band well above 1E-30 remains unclear to me. Did you figure out why that happens?

**Comment by mgrasso24** [comment]
> The question regarding the band well above 1E-30 remains unclear to me. Did you figure out why that happens?

Yes I added it to my fourth point in the conclusion "    The band on the eQTL data that is below 1E-30, is due to the restrictions on the program used in the Hudsonplot. Therefore, I want to clarify what that data showed. I looked at the data with a pvalue less than 1E-30 and it showed that they identify the pvalue being 0 and are displayed at the top of the graph (eQTL). "

> Thanks. Still the conclusion seems to be "everything is eQTL" and not particularly about your top variants in GWAS. Remember that I asked you to try different thresholds for eQTL until you think the message is clear. Could you try 1E-30 see what it looks like? There is not much of a difference between 1E-5 and 1E-7 so we dont have to show 1E-7 if you showed 1E-5 already.

I did different thresholds but the issue is that even with the threshold being 1E-30 there are 65,000 variants for eQTL.

**Comment by gaow** [comment]
> Yes I added it to my fourth point in the conclusion " The band on the eQTL data that is below 1E-30, is due to the restrictions on the program used in the Hudsonplot. Therefore, I want to clarify what that data showed. I looked at the data with a pvalue less than 1E-30 and it showed that they identify the pvalue being 0 and are displayed at the top of the graph (eQTL). "

thanks -- sorry I missed that earlier. 

> I did different thresholds but the issue is that even with the threshold being 1E-30 there are 65,000 variants for eQTL since the rest of the variants after that filter have a pvalue of 0

Okay let me clarify the key issue and motivation i would like you to do this hudson plot analysis -- you initally checked the data by overlapping the variants and only found a few variants (maybe 2?) overlap the interesting GWAS SNPs and eQTL data exactly. That small overlap definitely is not going to work for formal statistical analysis as our next step.

I believe that lack of overlap is not a lack of shared signal, but rather a lack of genotyping bias, that some gneotypes are simply not measured in these two data-sets. To show there is still sufficient signal we need to consider variants in LD with the ones of interest. To achieve that , simply plotting them seems the easiest thing to do. LD was written in your aim section but I think you still need to understand what exactly we try to achieve by leveraging LD ... Currently with your visualization of very many signals for the eQTL, it is entirely possible that the eQTLs near the GWAS SNPs have a very insignificant p-value but you cannot tell that from the plot because the eQTL panel is too busy. That's why I was asking to increase the filter hoping that the strong eQTLs will have a very small p-value in the neighborhood fo GWAS signals anyways and will show better when you clean it up via thresholding. Unfortuantely as you pointed out 65K variants is still too much ... 

Let me suggest another way: instead of doing this genome-wide, what if for each GWAS SNP < 1E-5 you add 100Kb in terms of position around it to subset the eQTL data and plot the subset of eQTLs? For example say your GWAS SNP is:

```
chr1  3135523
```

Then for this SNP you should get all SNPs in the eQTL data with coordinates between chr1:(3135523 - 100000) and chr1:(3135523 + 100000) then within that range you filter eQTL by p-value < 1E-5. Of course you can take the union of all the eQTLs selected this way and plot that against the GWAS data.  Human genome LD block on average should be less than 15Kb so 100Kb radius is pretty generous. You can shorten this to 50Kb too ... Does it make sense?

**Comment by gaow** [comment]
Of course, if we can establish there are potential eQTL signals in the "LD friends" of the GWAS data we can improve it in two ways: 1. get the latest GWAS summary stats from 23andme which should contain more GWAS variants (need to wait for the paper to come out) and 2. get the latest brain eQTL data from my lab which should have way more variants (need some work on that on our end). Now we are using existing public data to justify our proposed efforts with 1 and 2, through hudson plot visualization.

**Comment by mgrasso24** [comment]
> Does it make sense?

Yes, it does! Thank you for the explain and the advice on how to complete this! I will get on this right know.

**Comment by mgrasso24** [comment]
@gaow 
I was able to find some "LDfriends"! 

Therefore, I turned the LDfriends data frame into a Hudson plot with the GWAS. I updated everything to github, let me know if you have any comments or any conclusions to add based on the LDfriends Hudson plot.

**Comment by gaow** [comment]
Thanks @mgrasso24 this is now clearer. Please allow me to point out that the visual overlap between the two studies you see are likely due to LD but we don't know that yet for sure until we examined the LD data. Still, based on our past experience what you found should indeed be the "LD friends". Now from these plots we see that somewhat unfortunately the SNPs with strong GWAS signal do not see to have extremely strong eQTL (or LD friends of them); however the possible pleiotropic signals are still quite promising. 

Let's move on to the next step of the project: Bayesian colocalization. I"ll start it on a separate thread

**Comment by mgrasso24** [comment]
Ok, sounds good, I will do that in the future.

**Comment by mgrasso24** [comment]
Ok, sounds good, I will do that in the future.

**Comment by mgrasso24** [comment]
Ok, sounds good, I will do that in the future.
